### PR TITLE
Fixed Dashboard

### PR DESCRIPTION
**Pull request by LeeRenJie**
## Issue that this pull request solves

 Closes: #388 

## Proposed changes
- [x] For Dashboard, please do the following -
- Remove the count data
- Make the icons a bit bigger
- Make the content font a bit bigger(16px maybe)
- The card should contain only heading and content

- [ ] The Profile section has got distorted, please fix it

### Brief description of what is fixed or changed
1. Removed the count data
1. Bigger Icons
1. Content font 16px
1. Card contains only heading and content
 
## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
Before:
DBIconsBefore
After:
DBIconsAfter
Dashboard now:
Dashboard
Full Dashboard now:
FULLDASH

## Other information
Not sure which part is the profile distorted. 

Any other information that is important to this pull request

### CODE REVIEW

**Comment by Kajol-Kumari** [comment]
Hey @LeeRenJie On dashboard, the card's icon size needs to be increased. Also there is huge gap between title and content of the card, please reduce that and increase the font size of card content. Also in order to see the profile section click on the `Good Morning Admin` section of the Admin Dashboard.

**Comment by LeeRenJie** [comment]
> Hey @LeeRenJie On dashboard, the card's icon size needs to be increased. Also there is huge gap between title and content of the card, please reduce that and increase the font size of card content. Also in order to see the profile section click on the `Good Morning Admin` section of the Admin Dashboard.

Alright will work on it!

**Comment by LeeRenJie** [comment]
Profile:
image
Dashboard now:
dashboard

File: frontend/src/app.css [hunk 1]
```diff
@@ -21,9 +21,12 @@ body {
 }
 
 .content {
```
Comment (Kajol-Kumari) [inline frontend/src/app.css:23]: please avoid using global css wherever possible. If these styles are required somewhere, please put it in their respective css module else remove it

Comment (LeeRenJie) [reply frontend/src/app.css:23]: ok i'll try!

Comment (LeeRenJie) [reply frontend/src/app.css:23]: I can't seem to override the global attribute. I tried:
* .crd .content
* .crd > .content
* using !important in the styles

Is there anyway to tackle this?

**Review by Kajol-Kumari — APPROVED** [review]

**Pull request merged**